rpc/js/src/dial.ts

```diff
@@ -339,15 +353,15 @@ export async function dialWebRTC(
   // TODO(RSDK-2836): In general, this logic should be in parity with the golang implementation.
   // https://github.com/viamrobotics/goutils/blob/main/rpc/wrtc_client.go#L160-L175
   const config = await getOptionalWebRTCConfig(signalingAddress, host, opts);
```

Comment:
Looks like we're doing this to get additional ice servers from the machine. Why is this necessary?

Reply:
added a comment!